design-documents/service-isolation/split-extensions.md

```diff
@@ -61,20 +61,41 @@ Cons:
 
 ### New module - old composer
 
-This strategy proposes to expose all new modules via one (currently existing) `composer.json` and more described in [this proposal](https://github.com/magento/architecture/issues/88).
+This strategy proposes to expose all new modules via one (currently existing) `composer.json` and came from [this proposal](https://github.com/magento/architecture/issues/88).
 The main idea that extensions will use the same dependency in `composer.json` which will be distributed as metapackage. Magento core modules will depend on new modules like `catalog-ui` but not to `catalog` metapackage, such metapackage will be distributed separately.
+Such strategy assumes that existing modules will use the old namespaces but new modules will have the new namespaces.
 
  ```json
 "require": {
     "magento/module-catalog": "^103.0.0|^104.0.0"
 }
 ```
 
+Unlike the [original proposal]((https://github.com/magento/architecture/issues/88)), the `composer.json` probably will have another structure. As an example for `catalog` metapackage:
+
+```json
+"require": {
+    "magento/module-catalog-api": "^104.0.0",
```

Comment:
Why don't we depend on specific patch version here?

Reply:
The version here just for an example.